tasks/prune.rb

```diff
@@ -0,0 +1,20 @@
+#!/opt/puppetlabs/puppet/bin/ruby
+# frozen_string_literal: true
+
+require 'open3'
+
+require_relative '../../ruby_task_helper/files/task_helper'
+
+require_relative 'utils/application_factory'
+
+class ApplicationPruneTask < TaskHelper
+  def task(application:, environment:, keep:, **_kwargs)
+    ApplicationFactory.find(application, environment).each do |app|
+      app.prune(keep)
```

Comment:
`keep` What is it?
I suggest to use `keep:` named variable on `#prune`, to help understanding.